src/lib/bootloader/grub2_widgets.rb

```diff
@@ -339,6 +339,50 @@ def store
     end
   end
 
+  # Specific secure boot widget for ppc as we need to handle also
+  # secure boot state in hardware. For details see https://bugzilla.suse.com/show_bug.cgi?id=1206295
```

Comment:
So this is not testing secure boot widget on ppc, this is adding special secure boot widget for ppc. Again, very misleading commit message.

And what for?

What is the difference between the spacial ppc widget and the secure boot widget for other platforms?

It should be the same.

The only special behavior intended was that secure boot should be disabled by default on ppc when platform supports it but does not enable it.

Reply:
Well, the specific functionality of ppc widget is that it say in case that secure boot is enabled and not enabled in firmware, to display label with reminder to enabled secure boot also in hardware. See screenshots in pull request description. We do not do it for other architectures as e.g. on x86_64 you do not need to enable it, you can only enforce it in firmware, so it behaves differently ( at least what I understand so far ).

Reply:
How is that different? You enable/disable SB enforcement in firmware that supports it.

Reply:
Isn't that the same on x86? It either checks and enforces the signature, or does neither.

The IBM firmware actually has all options (do not care, check but not enforce, check and enforce) but the 'check but not enforce' option is disregarded, it's mostly for debugging or other special use.